docs/source/_static/css/custom.css

```diff
@@ -29,9 +29,9 @@ body {
 }
 
 img {
-    display: block;
+#    display: block;
     text-align: center;
-    margin: 2rem auto;
+#    margin: 2rem auto;
```

Comment:
Why this change? Now all images are left-aligned on the page instead of centred.

Reply:
Yes. This is to allow the translation status to be displayed on the main page: http://94.130.177.235/gha/
When looking around affected images I came to the conclusion that this looked better anyways.
I have unfortunately not been able to get the translation status properly aligned without this change.

Reply:
Let's revert these changes and give the <img> elements a CSS class of "inline". ... 

Add the following CSS rule underneath the above reverted rule:
```
img.inline {
    display: inline-block;
    margin: 0;
}
```

And specify the class in home.rst per my suggested change there.

This will style the translation images as you want while leaving the remainder of the images in the docs centred.